docs/user-guide/configuration/stages.md

```diff
@@ -0,0 +1,68 @@
+---
+layout: main
+title: Stages
+category: User Guide
+menu: menu
+toc:
+  - title: Example
+    url: "#example"
+  - title: Setup and Teardown
+    url: "#setup-and-teardown"
+  - title: Limitations
+    url: "#limitations"
+---
+
+# Pipeline Stages
+
+The configuration of stages is employed to categorize one or more jobs that execute steps to achieve a similar objective. For instance, jobs such as `ci-deploy`, `ci-test`, and `ci-certify` can be grouped together under a stage named integration.
+
+#### Example
+
+```
+jobs:
+  ci-deploy:
+    requires: []
+  ci-test:
+    requires: [ci-deploy]
+  ci-certify:
+    requires: [ci-test]
+  triggered-by-stage:
+    requires: [~stage@integration]
+
+stages:
+  integration:
+    requires: [~commit]
+    jobs: [ci-deploy, ci-test, ci-certify]
+    description: "This stage will deploy the latest application to the CI environment and certify it after the tests have passed."
+```
+
+<br>In the provided example, the `ci-deploy` job within the `integration` stage has an empty `requires` field, indicating that the job begins execution immediately upon stage activation, as demonstrated by the `commit` trigger. Conversely, the `triggered-by-stage` job, which is triggered by the integration stage, starts only after the completion of the last job or jobs within the stage.
+
+<br>Based on the configuration, the screenshot below illustrates the expected layout of the workflow graph.
```

Comment:
Some wording tweaks. Removed `<br>`, I don't think we need that. And added a sentence explicitly calling out the expected stage trigger syntax.

```suggestion
In the provided example, the `ci-deploy` job within the `integration` stage has an empty `requires` field, indicating that the job begins execution immediately upon stage activation, as demonstrated by the `commit` trigger. Conversely, the `triggered-by-stage` job, which is triggered by the `integration` stage, starts only after the completion of the last job or jobs within the stage. To configure a job that is triggered by the end of a stage, use the syntax `~stage@stageName`.

Based on the configuration, the screenshot below illustrates the expected layout of the workflow graph.
```